libs/platform/src/lib/components/form/combobox/combobox/combobox.component.ts

```diff
@@ -89,7 +84,7 @@ export class ComboboxComponent extends BaseCombobox implements OnInit, AfterView
 
     /** @hidden */
     ngOnInit(): void {
-        const providers = this.providers.size === 0 ? this._comboboxConfig.providers : this.providers;
+        const providers = this.providers && this.providers.size === 0 ? this._comboboxConfig.providers : this.providers;
```

Comment:
Since we a using TS greater than 3.7 it could be simplified:
```suggestion
        const providers = this.providers?.size === 0 ? this._comboboxConfig.providers : this.providers;
```